core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4114,8 +4123,10 @@ private static String javadocLink(String fullyQualifiedClassname) {
         "alluxio.user.block.write.location.policy.class";
     public static final String USER_CONF_CLUSTER_DEFAULT_ENABLED =
         "alluxio.user.conf.cluster.default.enabled";
-    public static final String USER_CONF_VERSION_SYNC_INTERVAL =
-        "alluxio.user.conf.version.sync.interval";
+    public static final String USER_CONF_HASH_SYNC_INTERVAL =
+        "alluxio.user.conf.hash.sync.interval";
```

Comment:
I think "hash" part is a bit too explicit, and we can do without it.
For user facing values, just "conf sync" should be OK.